config.template/faf-aio-replayserver/faf-aio-replayserver.yml

```diff
@@ -1,21 +0,0 @@
-rs:
-    log_level: INFO
-    server:
-        port: "15001"
-        prometheus_port: "8011"
-        connection_header_read_timeout: "21600"
-    db:
-        host: faf-db
-        port: "3306"
-        user: faf-aio-replayserver
-        name: faf
-    storage:
-        vault_path: /content/faf/vault/replay_vault
-    replay:
-        forced_end_time: "18000"
-        grace_period: "30"
-        delay:
-            replay_delay: "300"
-            update_interval: "1"
-        merge:
-            desired_quorum: "2"
```

Comment:
Mazornoob won't like you :-D

Reply:
I think, you'll need to do:
`git checkout develop`
`git pull`
`git checkout -`
`git rebase develop`

Reply:
I think, you'll need to do:
`git checkout develop`
`git pull`
`git checkout -`
`git rebase develop`

Reply:
Yeah that rebase went horribly wrong.